src/main/resources/assets/jade/lang/zh_tw.json

```diff
@@ -10,15 +12,27 @@
   "tooltip.jade.state_on": "開啟",
   "tooltip.jade.state_off": "關閉",
   "tooltip.jade.power": "能量：%d",
+  "jade.input_signal": "輸入：%d",
   "tooltip.jade.villager_profession": "職業：%s",
+  "gui.jade.by": "Snownee 使用 %s 製作",
+  "gui.jade.configuration": "Jade 設置",
+  "gui.jade.configuration.desc1": "一個§f使用者介面增強§r模組，",
+  "gui.jade.configuration.desc2": "顯示有關§f你正在查看§r的東西的§f資訊§r。",
```

Comment:
It is recommended to keep only one line and leave `gui.jade.configuration.desc2` empty, because scaled CJK characters are difficult to read.